spanner/spanner_snippets/snippet_test.go

```diff
@@ -208,4 +208,38 @@ func TestSample(t *testing.T) {
 
 	out = runCommand(t, "dmlbatchupdate", dbName)
 	assertContains(out, "Executed 2 SQL statements using Batch DML.")
+
+	out = runCommand(t, "createtablewithdatatypes", dbName)
```

Comment:
Can this be moved to a different test function? This one is really long and it's not clear it all needs to be the same test.

If all of the datatypes queries can be run in parallel, we can do that, too. The spanner tests are one of the slowest in the repo so I don't want to make it worse.

Running in parallel would look something like https://github.com/GoogleCloudPlatform/golang-samples/blob/master/dlp/dlp_snippets/inspect_test.go#L35 (need to convert to a table test (`[]struct{...}{...}`), use `t.Run`, and use `t.Parallel`.

Reply:
I'd prefer to leave this as is for simplicity.
These additional tests are simple and run quickly so the
time added to run these with the existing tests should be
negligible. This can be revisited in a future update.